### PR TITLE
Support quay.io registry

### PR DESCRIPTION
**Pull request by phracek**
This commit adds support for the quay.io registry. Docker.io implements a rate limit which is a pain.

### CODE REVIEW

**Review by frenzymadness — CHANGES_REQUESTED** [review]
Good job! But, the last commit is not complete. It should contain also a new version of Dockerfiles for 2.7 and 3.8.

By the way, are we moving the images or just creating copies for our CI?

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
@frenzymadness Updates are done. Let's wait for the tests.

[test]

**Comment by phracek** [comment]
@frenzymadness Tests are green. After review, we can merge it.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Python container images
-====================
+=======================
+s2i-python-container 2.7 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/python-27-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/python-27-centos7)
+
+s2i-python-container 3.8 Quay.io status: [![Docker Repository on Quay](https://quay.io/repository/centos7/python-38-centos7/status "Docker Repository on Quay")](https://quay.io/repository/centos7/python-38-centos7)
```
Comment (frenzymadness) [inline README.md:5]: One last thing, could we make this on one line?

Comment (phracek) [reply README.md:5]: s2i-python-container 2.7 [status], s2i-python-container 3.8 [status]?

Comment (frenzymadness) [reply README.md:5]: Yes, is it possible?

Comment (phracek) [reply README.md:5]: Sure

**Review by frenzymadness — APPROVED** [review]

**Comment by frenzymadness** [comment]
Thank you!

**Pull request merged**